perses/charts/values.yaml

```diff
@@ -154,7 +154,7 @@ perses:
     provisioning:
       folders:
         - /etc/perses/provisioning
-      interval: 10s
+      # interval: 10m
```

Comment:
We can rely on the upstream interval :) 
https://github.com/perses/helm-charts/pull/80/files